src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.service.FileRead;
+import core.basesyntax.service.FileWrite;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.impl.DataConverterImpl;
+import core.basesyntax.service.impl.FileReadImpl;
+import core.basesyntax.service.impl.FileWriteImpl;
+import core.basesyntax.service.impl.ReportGeneratorImpl;
+import core.basesyntax.service.impl.ShopServiceImpl;
+import core.basesyntax.stategy.BalanceHandler;
+import core.basesyntax.stategy.FruitOperationHandler;
+import core.basesyntax.stategy.OperationStrategy;
+import core.basesyntax.stategy.OperationStrategyImpl;
+import core.basesyntax.stategy.PurchaseOperation;
+import core.basesyntax.stategy.ReturnOperation;
+import core.basesyntax.stategy.SupplyOperation;
+import core.basesyntax.validator.DataValidator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final DataValidator validator = new DataValidator();
+    private static final Map<String, Integer> inventory = new HashMap<>();
+    private static final FileRead fileRead = new FileReadImpl();
+    private static final DataConverter dataConverter = new DataConverterImpl(validator);
```

Comment:
Ensure that the `DataConverterImpl` class correctly implements the `DataConverter` interface and handles any potential exceptions during data conversion.